studio/schemas/documents/admin/legalDocuments.ts

```diff
@@ -1,18 +1,35 @@
 import { defineField } from "sanity";
 
+import languageSchemaField from "internationalization/languageSchemaField";
+import { supportedLanguages } from "internationalization/supportedLanguages";
 import { richText, title } from "studio/schemas/fields/text";
 import { titleSlug } from "studio/schemas/schemaTypes/slug";
 
-export const legalDocumentID = "legalDocuments";
+export const legalDocumentID = "legalDocument";
 
 const legalDocument = defineField({
   name: legalDocumentID,
   type: "document",
   title: "Legal Document",
-  fields: [title, titleSlug, richText],
+  fields: [languageSchemaField, title, titleSlug, richText],
   preview: {
     select: {
       title: "basicTitle",
+      language: "language",
+    },
+    prepare({ title, language }) {
+      const languageEntry = supportedLanguages.find(
```

Comment:
Could be nice to extract this to a common function (e.g. `getLanguageById`) in `supportedLanguages.ts`